colmi_r02_client/client.py

```diff
@@ -16,7 +16,12 @@
     set_time,
     blink_twice,
     hr,
-    hr_settings,
+    hr_settings
+)
+
+from colmi_r02_client.real_time import (
+    enum as rt_enum,
+    packet as rt_packet
```

Comment:
minor nit, I don't like renaming imports. I'd import `real_time` and refer to it as `real_time.enum` and `real_time.packet`